src/workspacer/workspacer.csproj

```diff
@@ -27,7 +27,8 @@
     <ProjectReference Include="..\workspacer.Gap\workspacer.Gap.csproj" ReferenceOutputAssembly="false" />
     <ProjectReference Include="..\workspacer.ActionMenu\workspacer.ActionMenu.csproj" ReferenceOutputAssembly="false" />
     <ProjectReference Include="..\workspacer.FocusIndicator\workspacer.FocusIndicator.csproj" ReferenceOutputAssembly="false" />
-    <ProjectReference Include="..\workspacer.TitleBar\workspacer.TitleBar.csproj" ReferenceOutputAssembly="false" />
+    <ProjectReference Include="..\workspacer.Sound\workspacer.Sound.csproj" />
```

Comment:
Just checking if this is deliberate or not... In this case we *want* to reference the output assembly, right?